bucket/go-jsonnet.json

```diff
@@ -0,0 +1,17 @@
+{
+    "version": "0.17.0",
+    "description": "A data templating language for app and tool developers",
+    "homepage": "https://github.com/google/go-jsonnet",
+    "license": "Apache License 2.0",
+    "url": "https://github.com/google/go-jsonnet/releases/download/v0.17.0/go-jsonnet_0.17.0_Windows_x86_64.tar.gz",
+    "hash": "d4d56a4e81752b53b301d92066678d036571224383da268896149c087e5a3c03",
+    "extract_dir": "go-jsonnet_0.17.0_Windows_x86_64",
+    "bin": [
+        "jsonnet.exe",
+        "jsonnetfmt.exe"
+    ],
+    "checkver": "github",
+    "autoupdate": {
```

Comment:
Add hash extraction from checksums.txt

Reply:
Added without regex, as it seemed to be what other Go packages are doing (e.g. lazygit)